changelog/_unreleased/2021-11-26-hardened-criteria-evaluation.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Made the evaluation of Criteria object more robust.
+issue: NEXT-19034
+flag:
+author: Ulrich Thomas Gabor
+author_email: ulrich.thomas.gabor@odd-solutions.de
+author_github: @UlrichThomasGabor
+---
+# Core
+* Made the evaluation of Criteria object more robust, i.e. it is not possible anymore to pass in non-existing associations.
```

Comment:
can you update the changelog to reflect the changes of the solution (Log instead of exception)

Reply:
@UlrichThomasGabor Can you update it?

Reply:
@shyim Yes, will be done till Monday I think